examples/coexist_klog_v1/coexist_klog_v1.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"flag"
+
+	"k8s.io/klog"
+	klogv2 "k8s.io/klog/v2"
+)
+
+type klogWriter struct {}
+func (kw klogWriter)Write(p []byte) (n int, err error){
+	klog.Info(string(p))
```

Comment:
doesn't this misattribute line numbers, mess up vmodule, and lose info/warn/fatal distinctions?